test/fork/utils/TwoPhaseFrameConfigUpdate.t.sol

```diff
@@ -0,0 +1,320 @@
+// SPDX-FileCopyrightText: 2025 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.24;
+
+import { Test } from "forge-std/Test.sol";
+import { DeploymentFixtures } from "../../helpers/Fixtures.sol";
+import { Utilities } from "../../helpers/Utilities.sol";
+import { TwoPhaseFrameConfigUpdate } from "../../../src/utils/TwoPhaseFrameConfigUpdate.sol";
+
+contract TwoPhaseFrameConfigUpdateTest is Test, Utilities, DeploymentFixtures {
```

Comment:
I think it should be a part of integration suite with no need for additional just command